src/popup/components/TransferSend/TransferSendRecipient.vue

```diff
@@ -58,25 +59,29 @@ import {
 } from 'vue';
 import { Field } from 'vee-validate';
 
-import type { Protocol, IInputMessage } from '@/types';
+import {
+  Protocol, IInputMessage,
+} from '@/types';
 import { getMessageByFieldName } from '@/utils';
-import { MODAL_ADDRESS_BOOK_ACCOUNT_SELECTOR, MODAL_RECIPIENT_INFO, PROTOCOLS } from '@/constants';
+import {
+  MODAL_ADDRESS_BOOK_ACCOUNT_SELECTOR, MODAL_RECIPIENT_INFO, PROTOCOLS,
```

Comment:
We have this unwritten rule that if the EsLint warns you about too many items in one line we split all items into separate lines.